scrapers/Staxus.yml

```diff
@@ -0,0 +1,382 @@
+name: "Staxus"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://staxus.com/trial/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://staxus.com/trial/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="text"]
+    scene:
+      Title:
+        selector: //title/text()
+        postProcess:
+          - replace:
+              - regex: 'Staxus Scene - '
+                with: ""
+      Date:
+        selector: //div[@class='video-details'][1]
+        postProcess:
+          - replace:
+              - regex: "Added: "
+                with: ""
+          - parseDate: 02/January/2006
+      Tags:
+        Name: //p/a[@href[contains(., "category.php")]]/text()
+      Performers:
+        Name:
+          selector: //div[@class="video-descr__model-item"]/p/a/text()
+      Details:
+        selector: //div[@class="video-descr__content"]/p/text()
+        concat: "\n"
+      Image:
+        selector:  //script[@type="application/ld+json"][contains(.,'"thumbnailUrl":')][1]
+        postProcess:
+          - replace:
+              - regex: '^.*thumbnailUrl": "'
+                with: ""
+              - regex: '.jpg.*$'
+                with: ".jpg"
+              - regex: 'contentthumbs'
+                with: "content/contentthumbs"
```

Comment:
I think there is a single script with the selector used so `[1]` is not needed
We can also use a different regex to remove a replace
```yaml
      Image:
        selector: //script[@type="application/ld+json"][contains(.,'"thumbnailUrl":')]
        postProcess:
          - replace:
              - regex: '^.*thumbnailUrl":\s"([^"]+)".*$'
                with: "$1"
              - regex: "contentthumbs"
                with: "content/contentthumbs"
```